src/service/resource.js

```diff
@@ -1093,6 +1093,9 @@ export class Resource {
     if (this.element.unlayoutOnPause()) {
       this.unlayout();
     }
+    if (this.element.V1()) {
+      this.element.pause();
```

Comment:
Shouldn't we just always call `pause` and remove the `pauseCallback` path above?

Reply:
Yeah. I see your point. This kind of looks recursive. The issue has been that it's rather dangerous to reorder `Resource -> CustomElement` calls. E.g. recall the problem with build state being tracked in both CE and Resource. So I've been trying to keep the old `Resource -> CE` path completely untouched. I think it'd be safer to update all the call sites instead.

Reply:
Refactored it a bit. Now the calls are in the same direction. The `Resource.pause()` simply calls `Element.pause()`, which decides what to do based on `V1`. A similar story with `Resource.unload() -> Element.unmount()`. I agree - this looks better.